docs/VehicleInfo/GetVehicleData/index.md

```diff
@@ -9,6 +9,20 @@ Sender
 Purpose
 : Get current values of specified vehicle data types.
 
+!!! must
+1. Consider a parameter in GetVehicleData request as `name` in case it is listed in HMI_API.xml; `name` is always boolean (according to API).  
+2. Consider a parameter in GetVehicleData request as `key` in case it is not listed in HMI_API.xml; `key` in GetVehicleData request may be either boolean or struct. Struct may contain either booleans or structs. 
```

Comment:
```suggestion
2. Map each parameter in GetVehicleData request to its `<key>` in the case that it is not listed in HMI_API.xml; `<key>` in GetVehicleData request may be either a boolean or struct value. If the value is a struct, it may contain either boolean or nested struct values, structured according to the vehicle data structure defined in `schema_items`. 
```